=== tests/cases/compiler/sourceMapValidationTryCatchFinally.ts ===
var x = 10;
>x : number
>类型别名 :=> 数字
>10 : 10

try {
    x = x + 1;
>x = x + 1 : number
>类型别名 :=> 数字
>x : number
>类型别名 :=> 数字
>x + 1 : number
>类型别名 :=> 数字
>x : number
>类型别名 :=> 数字
>1 : 1

} catch (e) {
>e : any
>类型别名 :=> 任意

    x = x - 1;
>x = x - 1 : number
>类型别名 :=> 数字
>x : number
>类型别名 :=> 数字
>x - 1 : number
>类型别名 :=> 数字
>x : number
>类型别名 :=> 数字
>1 : 1

} finally {
    x = x * 10;
>x = x * 10 : number
>类型别名 :=> 数字
>x : number
>类型别名 :=> 数字
>x * 10 : number
>类型别名 :=> 数字
>x : number
>类型别名 :=> 数字
>10 : 10
}
try
{
    x = x + 1;
>x = x + 1 : number
>类型别名 :=> 数字
>x : number
>类型别名 :=> 数字
>x + 1 : number
>类型别名 :=> 数字
>x : number
>类型别名 :=> 数字
>1 : 1

    throw new Error();
>new Error() : Error
>Error : ErrorConstructor
}
catch (e)
>e : any
>类型别名 :=> 任意
{
    x = x - 1;
>x = x - 1 : number
>类型别名 :=> 数字
>x : number
>类型别名 :=> 数字
>x - 1 : number
>类型别名 :=> 数字
>x : number
>类型别名 :=> 数字
>1 : 1
}
finally
{
    x = x * 10;
>x = x * 10 : number
>类型别名 :=> 数字
>x : number
>类型别名 :=> 数字
>x * 10 : number
>类型别名 :=> 数字
>x : number
>类型别名 :=> 数字
>10 : 10
}
